cartsquad/apps/accounts/views.py

```diff
@@ -68,7 +68,7 @@ def user_login_view(request):
 def retailer_login_view(request):
     user = request.user
     if user.is_authenticated:
-        return redirect('homepage')
+        return redirect('/products/view_all')
```

Comment:
Upon sign in of retailers they should be navigated to page where they view all of their products